src/components/jurisdiction/Jurisdiction.vue

```diff
@@ -41,15 +42,26 @@ export default class Jurisdiction extends Vue {
         separator: (jur.value === JurisdictionLocation.FD)
       }))
 
+    if (this.showUsJurisdictions) {
+      array.push({ isHeader: true, group: 1, text: 'United States' })
+      UsaJurisdiction
+        .forEach(jur => array.push({
+          group: 1,
+          text: jur.text,
+          value: jur.value,
+          separator: false
+        }))
+    }
+
```

Comment:
Yui confirmed to about Puerto Rico and VI: "Please use the same as live Name Request. I.e. Puerto Rico and VI are considered International."